config/environments/production.rb

```diff
@@ -42,7 +42,7 @@
   elsif ENV['LAGO_REDIS_CACHE_URL'].present?
     cache_store_config = {
       url: ENV['LAGO_REDIS_CACHE_URL'],
-      error_handler: ->(method:, returning:, exception:) {
+      error_handler: lambda { |method:, returning:, exception:|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `method`. If it's necessary, use `_` or `_method` as an argument name to indicate that it won't be used. (https://rubystyle.guide#underscore-unused-vars)